resolwe_bio/tests/workflows/test_wes.py

```diff
@@ -93,3 +97,10 @@ def test_wes(self):
         self.run_process('workflow-wes', input_workflow)
         wes = Data.objects.last()
         self.assertFileExists(wes, 'vcf')
+
+        # Test skipping bamclipper
+        input_workflow_skipbc = input_workflow
+        input_workflow_skipbc['advanced']['bamclipper']['skip'] = True
+        self.run_process('workflow-wes', input_workflow_skipbc)
+        wes_skip = Data.objects.filter(process_info=['Skipping bamclipper step.'])
```

Comment:
It would be better to filter for the correct object based on process type, and then check if the correct process info is present.

Reply:
`.get` doesn't work because there are two objects available which throws an error. I use `filter()` and test for the last (highest ID) object.